en/docs/guides/mfa/x509.md

```diff
@@ -0,0 +1,400 @@
+        ```
+
+        Make sure to use the same password you used when creating the keystore for the `srcstorepass` in the above step. Now you have the `localhost.p12` file that you can import into your browser as explained in the [import certificate](#import-certificate) section.
+
+7.  Next, create a new trust store and import the server certificate into the trust store using the following commands:
+
+    ``` xml
+    keytool -import -keystore cacerts.jks -storepass cacertspassword -alias rootCA -file rootCA.crt -noprompt
+    keytool -importcert -alias localcrt -file localcrt.crt -keystore cacerts.jks -storepass cacertspassword -noprompt
+    ```
+
+    !!! tip "CN"
+        The User objects in the LDAP directory hierarchy have designators that start with CN, meaning Common Name. The CN designator applies to all but a few object types. Active Directory only uses two other object designators (although LDAP defines several).
+
+Once you have done the above steps, you have the keystore (`localcrt.jks`), truststore (`cacerts.jks`), and pkcs12 (`localhost.p12`) files that you need to use later on in this guide.
+
+---
+
+## Configure the X509 certificate for the app
+
+1.  Download the [WSO2 Identity Server](http://wso2.com/products/identity-server/).
+
+2.  Replace your keystore file path, keystore password, trust store file path and trust store password (you can use the keystore and
+    truststore, which you created in the [Working with Certificates](#working-with-certificates) section) in the following configuration and add it to the
+    `<IS_HOME>/repository/conf/deployment.toml` file.
+
+    ``` toml 
+    [custom_trasport.x509.properties]
+    protocols="HTTP/1.1"
+    port="8443"
+    maxThreads="200"
+    scheme="https"
+    secure=true
+    SSLEnabled=true
+    keystoreFile="/path/to/keystore.jks"
+    keystorePass="keystorepwd"
+    truststoreFile="/path/to/truststore.jks"
+    truststorePass="truststorespassword"
+    bindOnInit=false
+    clientAuth="want"
+    ssl_protocol = "TLS"
+    ```
+
+    !!! note
+    
+        1.   To function properly, this connector should come first in the order. Otherwise, when mutual SSL takes place, the already existing connector (9443) will be picked up and the certificate will not be retrieved correctly.
+
+        2.  The `clientAuth` attribute causes the Tomcat to require the client with providing a certificate that can be configured as follows.
+
+            -   `true` : valid client certificate required for a connection to succeed
+            -   `want` : use a certificate if available, but still connect if no certificate is available
+            -   `false` : no client certificate is required or validated
+    
+        -   The `truststoreFile` attributes specifies the location of the truststore that contains the trusted certificate issuers.
```

Comment:
truststore --> trust store

Please ensure this is consistent throughout the guide